src/Pdf.php

```diff
@@ -12,38 +13,64 @@ class Pdf
 
     protected $binPath;
 
+    protected $options = [];
+
     public function __construct(string $binPath = null)
     {
         $this->binPath = $binPath ?? '/usr/bin/pdftotext';
     }
 
-    public function setPdf(string $pdf) : Pdf
+    public function setPdf(string $pdf) : self
     {
-        if (!file_exists($pdf)) {
-            throw new PdfNotFound("could not find pdf {$pdf}");
+        if (!\is_readable($pdf)) {
```

Comment:
I'm not to keen on that `\` for ensuring the global functions are used. Could you remove all `\` prefixes you've used in function calls?